test/k8s-integration/cluster.go

```diff
@@ -150,9 +150,19 @@ func clusterUpGKE(gceZone, gceRegion string, numNodes int, imageType string) err
 			return err
 		}
 	}
-	cmd := exec.Command("gcloud", "container", "clusters", "create", gkeTestClusterName,
-		locationArg, locationVal, "--cluster-version", *gkeClusterVer, "--num-nodes", strconv.Itoa(numNodes),
-		"--quiet", "--machine-type", "n1-standard-2", "--image-type", imageType)
+
+	var cmd *exec.Cmd
+	if useManagedDriver {
+		// PD CSI Driver add on is enabled only in gcloud beta.
+		cmd = exec.Command("gcloud", "beta", "container", "clusters", "create", gkeTestClusterName,
```

Comment:
You should put the args in an array and then pass to Command as Command(args...)

That way all the common arguments only have to be stated once which will help maintenance long-term.

Reply:
Ack